userscript.config.js

```diff
@@ -0,0 +1,23 @@
+// ==UserScript==
+// @name            Simple YouTube Age Restriction Bypass
+// @description     Watch age restricted videos on YouTube without login and without age verification :)
+// @description:de  Schaue YouTube Videos mit Altersbeschränkungen ohne Anmeldung und ohne dein Alter zu bestätigen :)
+// @description:fr  Regardez des vidéos YouTube avec des restrictions d'âge sans vous inscrire et sans confirmer votre âge :)
+// @description:it  Guarda i video con restrizioni di età su YouTube senza login e senza verifica dell'età :)
+// @version         2.1.4
+// @author          Zerody (https://github.com/zerodytrash)
+// @namespace       https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/
+// @updateURL       https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/raw/main/Simple-YouTube-Age-Restriction-Bypass.user.js
+// @downloadURL     https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/raw/main/Simple-YouTube-Age-Restriction-Bypass.user.js
```

Comment:
These are not gonna work since the file does not exist on the repo anymore. Do we even need it? I read somewhere that the update process is handled automatically, because it checks for updates on the link you originally downloaded it from.

If that's not the case, this needs to point to the latest GitHub release `releases/latest/download/Simple-YouTube-Age-Restriction-Bypass.user.js`.

Reply:
https://stackoverflow.com/a/38025376

Reply:
My judgement is that we can remove both, because they are needed anyway.

Reply:
Okay, let's remove it.